python/foglamp/plugins/south/common/sensortag_cc2650.py

```diff
@@ -0,0 +1,453 @@
+        },
+        'configuration': {
+            'uuid': 'f000aa82-0451-4000-b000-000000000000',
+            'handle': '0x0000'
+        },
+        'period': {
+            'uuid': 'f000aa83-0451-4000-b000-000000000000',
+            'handle': '0x0000'
+        }
+    },
+    'humidity': {
+        'data': {
+            'uuid': 'f000aa21-0451-4000-b000-000000000000',
+            'handle': '0x0000'
+        },
+        'notification': {
+            'uuid': '2902',
+            'handle': '0x0000'
+        },
+        'configuration': {
+            'uuid': 'f000aa22-0451-4000-b000-000000000000',
+            'handle': '0x0000'
+        },
+        'period': {
+            'uuid': 'f000aa23-0451-4000-b000-000000000000',
+            'handle': '0x0000'
+        }
+    },
+    'pressure': {
+        'data': {
+            'uuid': 'f000aa41-0451-4000-b000-000000000000',
+            'handle': '0x0000'
+        },
+        'notification': {
+            'uuid': '2902',
+            'handle': '0x0000'
+        },
+        'configuration': {
+            'uuid': 'f000aa42-0451-4000-b000-000000000000',
+            'handle': '0x0000'
+        },
+        'period': {
+            'uuid': 'f000aa43-0451-4000-b000-000000000000',
+            'handle': '0x0000'
+        }
+    },
+    'luminance': {
+        'data': {
+            'uuid': 'f000aa71-0451-4000-b000-000000000000',
+            'handle': '0x0000'
+        },
+        'notification': {
+            'uuid': '2902',
+            'handle': '0x0000'
+        },
+        'configuration': {
+            'uuid': 'f000aa72-0451-4000-b000-000000000000',
+            'handle': '0x0000'
+        },
+        'period': {
+            'uuid': 'f000aa73-0451-4000-b000-000000000000',
+            'handle': '0x0000'
+        }
+    }
+}
+
+char_enable = '01'
+char_disable = '00'
+"""
+movement_enable setting
+Bits	Usage
+0	Gyroscope z axis enable
+1	Gyroscope y axis enable
+2	Gyroscope x axis enable
+3	Accelerometer z axis enable
+4	Accelerometer y axis enable
+5	Accelerometer x axis enable
+6	Magnetometer enable (all axes)
+7	Wake-On-Motion Enable
+8:9	Accelerometer range (0=2G, 1=4G, 2=8G, 3=16G)
+10:15	Not used
+"""
+movement_enable = 'FFFF'
+movement_disable = '0000'
+
+
+class SensorTagCC2650(object):
+    """Handles readings from SensorTagCC2650
+    """
+    reading_iterations = 1  # number of iterations to read data from the TAG
+
+    def __init__(self, bluetooth_adr):
+        try:
+            self.bluetooth_adr = bluetooth_adr
+            self.con = pexpect.spawn('gatttool -b ' + bluetooth_adr + ' --interactive')
+            self.con.expect('\[LE\]>', timeout=600)
+            print('SensorTagCC2650 {} Preparing to connect. Hold on a second...If nothing happens please press the power button...'.format(bluetooth_adr))
+            _LOGGER.info('SensorTagCC2650 {} Preparing to connect. Hold on a second...If nothing happens please press the power button...'.format(bluetooth_adr))
```

Comment:
too long, please break into lines. We should remove print statements. LOGGER is fine